src/clients/SpokePoolClient.ts

```diff
@@ -562,7 +544,7 @@ export class SpokePoolClient extends BaseAbstractClient {
     });
 
     const timerStart = Date.now();
-    const multicallFunctions = ["getCurrentTime", "numberOfDeposits"];
+    const multicallFunctions = ["getCurrentTime"];
```

Comment:
I left this intact multicall query in case there's anything else we want to query during update - it's easy to just append functions here. Is there likely to be anything else? If not we could alternatively just revert to a `SpokePool.getCurrentTime()` call.